modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -4616,6 +4623,13 @@ public CacheGroupDescriptor cacheGroupDescriptor(int grpId) {
      * @return Cache descriptor.
      */
     public @Nullable DynamicCacheDescriptor cacheDescriptor(int cacheId) {
+        Map<Integer, String> cacheIdToNameMapping = cacheIDToNameMapping();
+        String name = cacheIdToNameMapping.get(cacheId);
+
+        if (name != null) {
```

Comment:
As per our Coding Guidelines, {} not needed here.
https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines